### PR TITLE
Add submodule update issue template

### PR DESCRIPTION
**Pull request by lizziel**
### Name and Institution (Required)

Name: Lizzie Lundgren
Institution: Harvard University

### Confirm you have reviewed the following documentation

- [X] [Contributing guidelines](https://gchp.readthedocs.io/en/latest/reference/CONTRIBUTING.html)

### Describe the update

This PR adds back in the GCHP submodules update issue template. Bringing this template back is helpful for requesting multiple concurrent submodule PR merges. I added the requirement for name and institution.

### Expected changes

An additional template will be visible when creating a PR.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/submodule-updates.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<!--- Describe your update at a high level. For example: --->
+
+* _none_
+
+### Submodule Pull Requests
+
+<!--- Create a list of checkboxes (using "[ ]") for your submodule pull requests. --->
+<!--- If your pull requests should be merged in a specific order, please make sure they are ordered correctly. --->
+
+* [ ] https://github.com/geoschem/geos-chem/pull/834
```
Comment (yantosca) [inline .github/ISSUE_TEMPLATE/submodule-updates.md]: Should we make example numbers like pull/xxxx and note that we should replace then xxxx with the issue number?

**Review by yantosca — CHANGES_REQUESTED** [review]
Looks good.  I just had one question about the template checkboxes.

File: .github/ISSUE_TEMPLATE/submodule-updates.md (hunk 1 above)
Comment (lizziel) [reply .github/ISSUE_TEMPLATE/submodule-updates.md]: Will do.

**Review by yantosca — APPROVED** [review]
Good to merge

**Pull request merged**